fixtures/sponsors-admin.json

```diff
@@ -0,0 +1,4610 @@
+[
+{
+    "model": "sponsors.genericasset",
+    "pk": 1,
+    "fields": {
+        "polymorphic_ctype": 79,
+        "uuid": "b18bfddc-e6de-4d9f-977c-68c35c537de7",
+        "content_type": 60,
```

Comment:
Hm, just a word of warning that content_type ids are fairly unstable and prone to change.

I'd suggest trying to see if a dump using `--natural-primary --natural-foreign` can be re-loaded to avoid the inevitable headache of this fixture breaking when a new content type is introduced.